domain/src/main/java/org/oppia/android/domain/topic/TopicListController.kt

```diff
@@ -641,6 +647,9 @@ class TopicListController @Inject constructor(
         storyRecord.chaptersList.firstOrNull()?.let {
           nextChapterName = it.title
           explorationId = it.explorationId
+          // ChapterPlayState will be NOT_STARTED because this function only recommends the first
+          // story of un-started topics.
+          chapterPlayState = ChapterPlayState.NOT_STARTED
```

Comment:
Should be moved outside the ``let`` block since it's unrelated.

Reply:
moved outside `let`